packages/typeorm/src/__tests__/fixtures/connections/sqlite.connection.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import path from 'path';
+import {ConnectionOptions} from '../../../';
+import {Book} from '../entities/book.entity';
+
+export const SqliteConnection: ConnectionOptions = {
+  name: 'my-db',
+  type: 'sqlite',
+  database: path.join(__dirname, './mydb.sql'),
+  entities: [Book],
```

Comment:
I think it makes sense to have a booter for TypeORM entities too.

Reply:
That's the TypeORM convention, I think we should leave it alone. We just provide the glue, everything else - Entities, Connections, Respository methods - remain 100% pure TypeORM.